packages/orchestrator-ui-components/src/components/WFOTimeline/mapProcessStepStatusToEuiStepStatus.ts

```diff
@@ -0,0 +1,22 @@
+import { StepStatus } from '../../types';
+import { EuiStepStatus } from '@elastic/eui/src/components/steps/step_number';
+
+export const mapProcessStepStatusToEuiStepStatus = (
+    processStepStatus: StepStatus,
+): EuiStepStatus => {
+    switch (processStepStatus) {
+        case StepStatus.SUCCESS:
+        case StepStatus.SKIPPED:
+        case StepStatus.COMPLETE:
+            return 'complete';
```

Comment:
Does Eui expose an enum for the step statusses that you can use for the return value?